src/renderer/DummyDataframe.js

```diff
@@ -17,5 +17,9 @@ export default class DummyDataframe {
         this.metadata = metadata;
         this.propertyCount = 0;
         this._aabb = computeAABB(geom, type);
+        this.t1 = [0.1, 0.1, 0.1, 0.1];
```

Comment:
How about extracting the array to a descriptive variable? I'd also consider extracting these initializations to its own little methods. A reader can find quite difficult to understand these operations objective... what's `_aabb`, what's `t1`...`t4`?

Reply:
`_aabb` was long before this PR, and I don't think it's that bad, it's just the array of AxisAlignedBoundingBox, which is a common term in the context of graphics.

Regarding `t1`... `t4`, I agree it's very ugly. But the thing is that `t1`...`t4` are different things depending on the method are used (in all cases they are vectors, but that can be seen in the line too). Of course, to have multiple meanings it's even uglier, but this is a performance optimization that was added after discovering performance problems. They are used to re-use the memory of the vectors without re-allocating memory for new arrays. They are temporal variables that never escape the scope of the public methods of this class.

Reply:
Oh great, I didn't know AABB = AxisAlignedBoundingBox. Maybe we could add that somewhere in the comments, or even in the glossary.

Regarding to t1---t4, if they are just performance helpers, maybe we could add something like:
```js
ctor(){
   //... whatever
   this._initializeVectorsCache(); // or a similar name
}
```

and then something like
```js
/*
 * Initialize a set of variables to apply performance optimizations
 */
_initializeVectorsCache(){
  const seedValue = [0.1, 0.1, 0.1, 0.1];
  this.t1 = seedValue;
  this.t2 = seedValue;
  this.t3 = seedValue;
  this.t4 = seedValue;
}
```

That way the user is not distracted in the constructor but he gets a notion about them (t1...t4 are about performance, I won't worry about them for now)